src/Components/Classes/Table/index.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable prettier/prettier */
```

Comment:
remove this

Reply:
if i remove that line prettier formats the arrays in a way that eslint makes conflict